src/Context/SQLHandler.php

```diff
@@ -146,17 +146,15 @@ protected function requiredTableColumns($table)
         $table = trim($table, '`');
 
         // Statement to extract all required columns for a table.
-        $sqlStatement = "
-            SELECT 
-                column_name, data_type 
-            FROM 
-                information_schema.columns 
-            WHERE 
-                is_nullable = 'NO'
-            AND 
-                table_name = '%s'
-            AND 
-                table_schema = '%s';";
+        $sqlStatement = <<<SQL
+SELECT 
```

Comment:
While I don't have problem with this formatting, it is inconsistent with the rest of the files - can we format as above please?